pages/data-providers/[data-provider-id]/content.jsx

```diff
@@ -1,4 +1,5 @@
-import React, { useMemo } from 'react'
+// eslint-disable-next-line max-classes-per-file
```

Comment:
Prefer `/* eslint-disable max-classes-per-file */` in such cases. Why do you put it in the top?

Reply:
It's where my IDE puts it. Nonetheless it's gonna be removed in https://github.com/oacore/dashboard/pull/196